tools/bazel.rc

```diff
@@ -1,6 +1,7 @@
 # Envoy specific Bazel build/test options.
 
 build --workspace_status_command=bazel/get_workspace_status
+build --experimental_cc_skylark_api_enabled_packages=@rules_foreign_cc//tools/build_defs,tools/build_defs
```

Comment:
I think we might need to add  "@foreign_cc_impl" to this list; see https://github.com/bazelbuild/rules_foreign_cc/blob/master/README.md#rules_foreign_cc